### DISCUSSION TITLE
30級製革理符無法交任務(不用理會,是官方BUG)

### DISCUSSION THREAD

**Discussion by jsnjsn761** [GpointChen/FFXIVChnTextPatch-GP#221] [General]
![ffxiv_08262022_163318_658](https://user-images.githubusercontent.com/23568308/186859725-a2640946-4cd6-4a77-b31a-058eee86804c.png)

**Comment by jsnjsn761** [comment]
還原了也是一樣,應該是官方bug
與漢化無關?

**Comment by GpointChen** [comment]
![image](https://user-images.githubusercontent.com/34565875/187014051-45dfe07b-6394-48ba-b57f-34ca4da1ce22.png)

我是正常的，可能你原本的檔案就損毀了？有試過用release附的backup.rar還原看看嗎？

**Comment by jsnjsn761** [comment]
> ![image](https://user-images.githubusercontent.com/34565875/187014051-45dfe07b-6394-48ba-b57f-34ca4da1ce22.png)
> 
> 我是正常的，可能你原本的檔案就損毀了？有試過用release附的backup.rar還原看看嗎？

我重裝後也不能,轉語言也不能

**Comment by itsbosco1025** [comment]
我發現同一個NPC有一個以上的理符就會不行，應該是在選擇交哪一個理符的視窗出了問題，而且應該不止是這個理符的問題，薩雷安的理符也是一樣

**Comment by GpointChen** [comment]
https://www.reddit.com/r/ffxiv/comments/wwutkp/62_levequest_bug_solution_do_not_get_multiple/

看起來是官方問題沒錯，那麼我轉成Discussion了。
